packages/design-system/src/components/IdleTimeout/IdleTimeout.tsx

```diff
@@ -103,10 +103,10 @@ const IdleTimeout = ({
   const MS_BETWEEN_STATUS_CHECKS = 30000;
   // convert minutes to milliseconds
   const msToTimeout = timeToTimeout * 60000;
-  const msToWarning = (timeToTimeout - timeToWarning) * 60000;
+  const msToWarning = timeToWarning * 60000;
   const [checkStatusTime, setCheckStatusTime] = useState<number>(null);
   const [showWarning, setShowWarning] = useState<boolean>(false);
-  const [timeInWarning, setTimeInWarning] = useState<number>(timeToWarning);
+  const [timeInWarning, setTimeInWarning] = useState<number>(timeToTimeout - timeToWarning);
```

Comment:
I think we need to use the same ceiling function in all the places where we set this `timeInWarning`, because in the story, I see `1.5 minutes` in the message at first.